airbyte-cdk/python/unit_tests/sources/streams/test_streams_core.py

```diff
@@ -99,6 +99,35 @@ def test_supports_incremental_cursor_not_set():
     assert not test_stream.supports_incremental
 
 
+def test_namespace_set():
```

Comment:
we should also add tests to verify that `as_airbyte_stream` works as expected. See the tests above for an example

Reply:
i added the namespace property to the existing test, i hope that is okay? Otherwise, happy to repeat underneath.

Reply:
Looks great!